### PR TITLE
Documentation slate

### PR DESCRIPTION
**Pull request by alcalyn**
http://alcalyn.github.io/navitia _(Link will become_ `canaltp.github.io/navitia`_)_

[Slate](https://github.com/tripit/slate) doc is in a separated folder (`documentation/slate`) for now. Seen with @stifoon, it should replace `documentation/navitia.io`.

Format has been changed from **rst** to **markdown**.

When some changes are performed to the doc, there is a script to run to **build html files from markdown**, and these html files are pushed to the branch `gh-pages` (see the exemple branch: https://github.com/alcalyn/navitia/tree/gh-pages). Then, github will automatically update `canaltp.github.io/navitia`.

Todo:
- [ ] Report new changes to slate
- [x] Jenkins job
- [ ] Delete old doc (navitia.io directory)
- [x] Doc's doc
- [ ] Add suggestions made by @colinele
- [x] A lot of links are not "clickable"
- [x] The right column is not used : first step is to remove all tab but url and have every url at the right place in the column (maybe scrootchable : it's not clear what example is linked to what part of the doc)
- [x] A pass on all the doc should be done, as it is not usable as is, or less usable than the previous doc

### CODE REVIEW

**Comment by pbougue** [comment]
Neat!
I have 2 quick remarks:
- not sure it's usefull to have javascript example tab (but I might have missed something)
- it's not straightforward to understand what example refers to what part of the doc (http://alcalyn.github.io/navitia/?javascript#apis) but maybe the original doc is not really straightforward too.

File: documentation/slate/source/includes/index.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+
+Getting started
+---------------
+
+The easiest is probably to jump to [Examples](#examples) below.
+
+At some point you will want to read:
+
+- [public_transport](#public_transport)
```
Comment (antoine-de) [inline documentation/slate/source/includes/index.md]: broken link

File: documentation/slate/source/includes/index.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+- [integration](#integration)
```
Comment (antoine-de) [inline documentation/slate/source/includes/index.md]: broken link

**Comment by antoine-de** [comment]
I do agree with @pbougue , the examples on the right panel do not seems very straightforward.

I also think the example url should have a custom font so they can be a bit more visible.

On the whole process, should we put a git hook to run the html pages generation ? should we put it on a separate repository ?

Anyway good job, we're not far to have at long last a beautiful doc :heart_eyes_cat:

**Comment by kinnou02** [comment]
The javascript tab can be useful in a near future, it would be nice to add example in few language: curl, js and python seem the ones we should focus on.

**Comment by alcalyn** [comment]
> On the whole process, should we put a git hook to run the html pages generation ?

Sure, I think about a Jenkins job. But first, Slate **needs an orphan branch** named `gh-pages` where the html files will be generated.

> should we put it on a separate repository ?

Seen with @stifoon, because the current doc is in `navitia/documentation/navitia.io`, the Slate doc will replace it and live in navitia repo. But we can still change the repo.

**Comment by kinnou02** [comment]
skip ci

**Comment by alcalyn** [comment]
skip ci

**Comment by alcalyn** [comment]
> it's not straightforward to understand what example refers to what part of the doc

Yes, after reading the doc, I think that there is a work to do on the content itself to adapt it to a 3-column format.

I changed the language tabs, there is a plaintext URL tab, a cURL and a Python tab. Json outputs will be always visible (see http://alcalyn.github.io/navitia/?python#line-code).

**Comment by antoine-de** [comment]
I'm not convinced of the value a a curl/python tab.

The best would maybe to discuss it together with @stifoon and @colinele

**Comment by antoine-de** [comment]
@alcalyn @stifoon @colinele what's the status  on this ? I think we should discuss about it.

**Comment by alcalyn** [comment]
For me it is ready to merge once the content is adapted to the format

**Comment by antoine-de** [comment]
I think it's worth discussing about the update process, where to store the documentation, some formatting issues, ...

**Comment by alcalyn** [comment]
A scrootch has been added (example: http://alcalyn.github.io/navitia/#apis, ctrl+F "Type of the section.").

**Comment by stifoon** [comment]
Great ! (ctrl+F "Type of the section")

**Comment by antoine-de** [comment]
yeap it's nice!

**Comment by pbougue** [comment]
Nice add!
It's not clear whether its a scrootch or a link, though (would require a :arrow_down_small: I guess)

**Comment by alcalyn** [comment]
Yes it was planned

**Comment by pbougue** [comment]
more TODO:
- Add suggestions made by @colinele
- A lot of links are not "clickable"
- The right column is not used : first step is to remove all tab but url and have every url at the right place in the column (maybe scrootchable : it's not clear what example is linked to what part of the doc)
- A pass on all the doc should be done, as it is not usable as is, or less usable than the previous doc

@stifoon @alcalyn @antoine-de @colinele @nicorey2 : As no one seems to be motivated by that, we may close that PR and forget about it.

**Comment by alcalyn** [comment]
http://alcalyn.github.io/navitia updated with links and no languages tab

**Comment by alcalyn** [comment]
http://alcalyn.github.io/navitia updated with code put into the second column

**Comment by alcalyn** [comment]
http://alcalyn.github.io/navitia updated after a general pass

**Comment by pbougue** [comment]
@colinele Your suggestions are not added, we let you see with @alcalyn if this is necessary for this PR or if it might be done in another PR.
Poke @nicorey2

**Comment by antoine-de** [comment]
we need to remove the "The filter parameter" section, it's from an old doc

**Comment by antoine-de** [comment]
I think the `About` section is not relevant anymore

**Comment by pbougue** [comment]
Ok with @antoine-de (I saw it and wasn't sure about it).

We have some other small issues on the doc, but I suggest to regroup them in that JIRA: http://jira.canaltp.fr/browse/NAVIO-469
So we can merge that PR but not loose any trace of what work is left.

**Pull request merged**